tests/test_named.py

```diff
@@ -800,6 +800,34 @@ def test_named_index_access():
         h[0:10:20j, 0:5:10j, "hello", False, 5]
 
 
+@pytest.mark.xfail  # ToDo: NamedHist's proxy should work
+def test_named_proxy():
+    """
+        Test named proxy -- whether NamedHist proxy works properly.
+    """
+
+    h = NamedHist.Regular(10, 0, 1, name="x").fill(x=[0.5, 0.5])
+    assert h[0.5j] == 2
+
+    h = (
+        NamedHist()
+        .Regular(10, 0, 1, name="x")
+        .Regular(10, -1, 1, name="y")
+        .fill(x=[0.5, 0.5], y=[-0.2, 0.6])
+    )
+
+    assert h[0.5j, -0.2j] == h[bh.loc(0.5), bh.loc(0.6)] == 1
+
+    # add axes to existing histogram
+    with pytest.raises(Exception):
```

Comment:
I recommend putting the exact exception here if possible, just to ensure it doesn't change without us knowing.

Reply:
Yes, I agree. This will be a huge task, and I'll find some time to specialize in doing this.

Reply:
add ToDo item - https://github.com/scikit-hep/hist/pull/54/commits/3f23a6537befec4b8a2c0da00f0158aa196d48be